app/services/oneoffs/npq/migrate_declarations_between_statements.rb

```diff
@@ -111,6 +112,19 @@ def change_declaration_states_for_to_statement(to_statement, statement_line_item
       eligible_declarations.each { |declaration| service.call(declaration) }
     end
 
+    def make_payable_declaration_eligible_for_to_statement(to_statement, statement_line_items)
+      declarations = statement_line_items.map(&:participant_declaration).uniq
+      payable_declarations = declarations.select(&:payable?)
+
+      return if to_statement.payable? || to_statement.paid?
+      return unless payable_declarations.any?
+
+      record_info("Marking #{payable_declarations.size} payable declarations back as eligible for #{to_statement.name} statement")
+
+      payable_declarations.each { |declaration| DeclarationState.eligible!(declaration) }
+      statement_line_items.map(&:eligible!)
```

Comment:
can we only mark the payable declaration statement line items as eligible not all of them that we pass in?